### ISSUE TITLE
"Something went wrong" when trying to add a song to the queue

### ISSUE THREAD

**Issue by Yoyopizzacat** [jeroentvb/spicetify-power-bar#47]
**Describe the bug**
There is an error message when trying to add a song to the queue and nothing gets added.

**To Reproduce**
1. 'alt/option + space' to open powerbar
2. 'cmd + enter' to add song to queue

**Expected behavior**
The song gets added to queue.

**Screenshots**
![image](https://github.com/jeroentvb/spicetify-power-bar/assets/109506521/2f81d23b-0e67-460d-94c7-a284e4107657)

**Device info (please complete the following information):**
 - OS: [macOS 13.4.1]
Spotify for macOS (Apple Silicon)
1.2.17.834.g26ee1129
Spicetify v2.22.2

**Additional context**

**Comment by jeroentvb** [comment]
It sounds like this is related to #32. For some reason this error happens when you've not played anything yet, or haven't played anything in a while (while having Spotify open). It's caused by the Spotify API returning a 403, something I can't fix.

If this also happens outside of those cases, could you provide extended reproduction steps?

**Comment by oneeyedreaper** [comment]
Facing the same issue.
Tried reinstalling the extension, but that didn't resolve it. 
Ctrl + Enter gives out Something Went Wrong while playing the song. 
It was working some weeks ago, so the update may have broken something.

**Comment by jeroentvb** [comment]
Since it's working for me I can't do anything about this issue.

I either need an error log and/or accurate reproduction steps to be able to fix it.

**Comment by oneeyedreaper** [comment]
will ![this](https://github.com/jeroentvb/spicetify-power-bar/assets/53879034/9422fb58-1e2c-4fc3-b329-8302d10bbe50) help?

**Comment by jeroentvb** [comment]
I tested on both windows and macOs, and seem to only be able to trigger that error when not having been active for a while, or just launching spotify without having played anything, similar to #32.

It looks like Spotify de-registers or hasn't registered the spotify/spicetify session and doesn't know what session to use when adding an item to the queue. This could be fixable by adding `device_id` to the add to queue call, but I don't know what id to use. Any id I tried resulted in a 404. [Their endpoint](https://developer.spotify.com/documentation/web-api/reference/get-a-users-available-devices) for fetching the ids doesn't seem to work either..

There's also some related [stackoverflow posts](https://stackoverflow.com/questions/76958040/spotify-404-error-when-trying-to-add-songs-to-queue-using-spotify-api) and [forum issues](https://community.spotify.com/t5/Spotify-for-Developers/Add-Items-to-Playlist-404-Errors/td-p/5332623) about this issue.

So unfortunately it looks like this isn't something I'm able to fix. 😕
